zcash_client_sqlite/src/wallet.rs

```diff
@@ -2001,12 +2002,14 @@ pub(crate) fn get_wallet_summary<P: consensus::Parameters>(
                JOIN transactions t ON t.id_tx = transaction_id
                WHERE t.block IS NOT NULL -- the spending transaction is mined
                OR t.expiry_height IS NULL -- the spending tx will not expire
+               -- TODO(schell): a bug in select_spendable_notes was fixed by changing this to
+               -- `>= :summary_height`. Should that also happen here?
                OR t.expiry_height > :summary_height -- the spending tx is unexpired
```

Comment:
There's a separate bug here; this should be `:target_height`, not `:summary_height`. The expiry height is the last block in which the transaction can be mined, and so if the expiry height is equal to the target height, then the spending tx is unexpired, so yes this should be `>=`.